plugin/external/README.md

```diff
@@ -0,0 +1,78 @@
+# external
+
+## Name
+
+*external* - resolve ingress and load balance IPs from outside kubernetes clusters.
```

Comment:
Don't say "ingress" just say "load balancer and external IPs". Because "ingress" will imply L7 ingress to people, NOT the L3/L4 load balancer "ingress" IPs.

That's a completely different thing and would also be very cool to be able to manage, but not now...

Reply:
done